hudi-gcp/pom.xml

```diff
@@ -66,10 +66,6 @@ See https://github.com/GoogleCloudPlatform/cloud-opensource-java/wiki/The-Google
      <groupId>org.apache.logging.log4j</groupId>
      <artifactId>log4j-api</artifactId>
     </dependency>
-    <dependency>
-      <groupId>org.apache.logging.log4j</groupId>
-      <artifactId>log4j-core</artifactId>
-    </dependency>
     <dependency>
       <groupId>org.apache.logging.log4j</groupId>
       <artifactId>log4j-1.2-api</artifactId>
```

Comment:
Same here -- we don't need these ones anymore. Let's scan whole project and clean them up